### ISSUE TITLE
NaN value for training and validation loss 

### ISSUE THREAD

**Issue by kanginthaya** [sharathadavanne/multichannel-sed-crnn#8]
Hi,
I am trying to run the script sed.py on the DCASE 2017 Task 3 dataset as outlined in your readme.md file. However, I repeatedly get Nan values for training and validation loss. I successfully compiled feature.py and collected all the features in .npz files. Furthermore, I also tried reverting back to Keras 2.0.8 from the latest version, but the problem still continues. After I read some of the other issues, the model hyperparameters were tuned to the DCASE 2017 Task 3 dataset, right? If so, any idea why am I still facing this problem? 

Any help is greatly appreciated!

**Comment by kanginthaya** [comment]
Works fine when using the library versions mentioned in Readme. The code needs to be changed when using newer versions.
